volume/local/local.go

```diff
@@ -13,13 +13,15 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/docker/docker/daemon/graphdriver/quota"
```

Comment:
Should be moved out of graphdriver

Reply:
Yeah, I was wondering about that as well. What'd be an appropiate place for it?
I looked through the other paths, but nothing really seems fitting.

Reply:
Just docked/docker/quota ?